scripts/update-tasks.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+declare -r SCRIPT_PATH=$(readlink -f "$0")
+declare -r SCRIPT_DIR=$(cd $(dirname "$SCRIPT_PATH") && pwd)
+
+log() {
+    local level=$1; shift
+    echo -e "$level: $@"
+}
+
+
+err() {
+    log "ERROR" "$@" >&2
+}
+
+info() {
+    log "INFO" "$@"
+}
+
+die() {
+    local code=$1; shift
+    local msg="$@"; shift
+    err $msg
+    exit $code
+}
+
+usage() {
+  cat <<-EOF
+    USAGE:
+      $(basename $SCRIPT_PATH) DEST_DIR
+EOF
+
+}
+
+declare -r CATALOG_VERSION="release-v0.7"
```

Comment:
can it be an argument instead of static?

Reply:
good idea ..

Reply:
done